felix/bpf/ut/whitelist_test.go

```diff
@@ -134,7 +134,26 @@ func TestSkipRedirectPeerHostToVMWorkload(t *testing.T) {
 
 		// Approved by HEP
 		Expect(ctr.Data().A2B.Approved).To(BeTrue())
-		// NOt approved by WEP yet
+		// Not approved by WEP yet
+		Expect(ctr.Data().B2A.Approved).NotTo(BeTrue())
+		Expect(ctr.Flags() & v3.FlagNoRedirPeer).To(Equal(v3.FlagNoRedirPeer))
+	})
+
+	skbMark = 0
+	runBpfTest(t, "calico_from_workload_ep", nil, func(bpfrun bpfProgRunFn) {
```

Comment:
:+1:

Reply:
I think it will be better to reset the ct map here so that we can be sure ct entries are created with the skip_redirect flag